src/models/__init__.py

```diff
@@ -14,9 +14,11 @@
 
 
 def load_model(model_path: Path, data_path: Optional[Path] = None,
-               model_type: Optional[str] = None) -> Union[RecurrentNetwork,
-                                                          LinearNetwork,
-                                                          LinearRegression]:
+               model_type: Optional[str] = None,
+               device: Optional[str] = 'cpu') -> Union[RecurrentNetwork,
+                                                       LinearNetwork,
```

Comment:
do we need an assert statement to ensure it's only `cpu` or `gpu`?

Reply:
hmm if there are multiple gpus it could be a variety of devices (e.g. `cuda:1`).
PyTorch will throw up a fuss if the device is wrong, but it will know better than us.